src/modules/groups/groups.js

```diff
@@ -24,65 +22,29 @@ const getAccessToken = async () => {
         }
       }
     );
-    return { access_token };
+    return { accessToken: data?.access_token };
   } catch (error) {
     return { error };
   }
 };
 
-const accessToken = async (req, res) => {
-  const { access_token, error } = (await getAccessToken()) || {};
+router.get('/token', async (req, res) => {
+  const { accessToken, error } = (await getAccessToken()) || {};
   if (error) {
     return res.status(400).send({
```

Comment:
use res.sendResponse, i have created a response interceptor to have uniform response structure